source/shared/net/concepts/concepts-delegation.rst

```diff
@@ -8,14 +8,16 @@ Delegation
 
-Not everyone with CCD has the resources needed to run a baker. **Delegation** enables everyone to earn interest. Any party with CCD may delegate some of their capital to a baker. This increases the baker’s chance of baking the next block and getting rewards, which are then shared with the delegators. This is a non-custodial solution: when a party delegates an amount of CCD to a baker, the CCDs are not transferred to the baker and remain under the party's control; they are just considered part of the baker's stake for the proof-of-stake protocol. Staked CCDs, both for delegators and bakers, cannot be spent while staked. Unstaking CCDs is subject to a cool-down period.
+Not everyone with CCD has the resources needed to run a baker. :ref:`Delegation<glossary-delegate>` enables everyone to earn interest. Any party with CCD may delegate some of their capital to a baker. This increases the baker’s chance of baking the next block and getting rewards, which are then shared with the delegators. This is a non-custodial solution: when a party delegates an amount of CCD to a baker, the CCDs are not transferred to the baker and remain under the party's control; they are just considered part of the baker's stake for the proof-of-stake protocol. Staked CCDs, both for delegators and bakers, cannot be spent while staked. Unstaking CCDs is subject to a :ref:`cool-down period<glossary-cool-down-period>`.
 
 Pools and Rewards
 =================
 
-A baker opens a pool, and other CCD holders delegate some stake to this pool. The probability that the baker running the pool is chosen to bake the next block is then proportional to the total stake in the pool. Rewards are distributed to everyone in the pool proportional to their stake, and the baker takes a commission on the earnings of the delegators. For example, suppose that a pool has 10,000 CCD. And suppose that in a 24 hour period this pool gets 30 CCD as reward and the commission of the baker is 10%. Then a party with 1,000 CCD in the pool will receive a share of 3 CCD. If this party is a delegator, 0.3 CCD are paid as a commission to the baker and the other 2.7 CCD are kept.
+A baker opens a :ref:`pool<glossary-baker-pool>`, and other CCD holders delegate some stake to this pool. The probability that the baker running the pool is chosen to bake the next block is then proportional to the total stake in the pool. Rewards are distributed to everyone in the pool proportional to their stake, and the baker takes a commission on the earnings of the delegators. For example, suppose that a pool has 10,000 CCD. And suppose that in a 24 hour period this pool earns 30 CCD as reward and the commission of the baker is 10%. The baker receives 3 CCD as commission. The rest (27 CCD) is distributed to the members of the pool according to their relative stake in their pool. This means a party with 1,000 CCD in the pool will receive 2.7 CCD.
```

Comment:
This example might be simplified for clarity, but I just realised that
"The baker receives 3 CCD as commission"
is kinda wrong, unless we think of it like: he pays commission to himself. :thinking:

Reply:
This example is becoming very contentious. Mikael wanted it changed and this was the change Thomas recommended. @td202 Do you have any recommendation to the wording so that this becomes a non-issue?

Reply:
Otherwise, I am tempted to remove the example altogether.